src/utils/utils.js

```diff
@@ -7,9 +7,9 @@
 export function mixin(source) {
   const args = [...arguments];
   for (let i = 1; i < args.length; i++) {
-    for ( const key of Object.keys(args[i])) {
-      source[key] = args[i][key];
-    }
+    d3.entries(args[i]).forEach((p) => {
```

Comment:
Surely but I was doing minimum fixes for the tests. Stupid `for of` loop. I'm sure there's a better way without d3 but didn't want to get into changing away from original source.